lib/utils/build.js

```diff
@@ -10,7 +10,8 @@ function customPost (program, callback) {
   const directory = program.directory
   let customPostBuild
   try {
-    customPostBuild = require(`${directory}/post-build`)
+    const gatsbyNodeConfig = require(`${directory}/gatsby-node`)
+    customPostBuild = gatsbyNodeConfig.postBuild
   } catch (e) {
     if (e.code !== 'MODULE_NOT_FOUND') {
       console.log('Failed to load custom post build script. It will be skipped.')
```

Comment:
We actually want something here like https://github.com/gatsbyjs/gatsby/blob/6c29c6c81a9f08f1f9993bf0744dc6636fdba078/lib/utils/build-page/url-resolver.js#L11 so we can see the error. I'm not sure actually why the error wasn't being logged out here before.